### PR TITLE
docs(dashboard): improve dashboard documentation

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

This pr improves dashboard documentation

### Description

* Use docker to run prowler dashboard
* Add info about adding an own compliance to compliance page

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/tutorials/dashboard.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ Prowler allows you to run your own local dashboards using the csv outputs provid
 prowler dashboard
 ```
 
+To run Prowler local dashboard with docker, use: 
+
+```sh
+docker run -e dashboard=dashboard <prowler-image>
```
Comment (jfagoagas) [inline docs/tutorials/dashboard.md]: What is the purpose of `-e dashboard=dashboard`?

Comment (pedrooot) [reply docs/tutorials/dashboard.md]: Thanks for the catch! I´ve updated it

**Review by jfagoagas — APPROVED** [review]
🔝

File: docs/tutorials/dashboard.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ Prowler allows you to run your own local dashboards using the csv outputs provid
 prowler dashboard
 ```
 
+To run Prowler local dashboard with docker, use: 
+
+```sh
+docker run <prowler-image> dashboard
```
Comment (jfagoagas) [inline docs/tutorials/dashboard.md]:
```suggestion
docker run toniblyx/prowler:latest dashboard
```

File: docs/tutorials/dashboard.md [hunk 3]
```diff
@@ -26,6 +32,8 @@ This page shows all the info related to the compliance selected, you can apply m
 
 <img src="./img/dashboard/dashboard-compliance.png">
 
+To add your own compliance to compliance page, add a file with the compliance name (using `_` instead of `.`) to the path `/dashboard/compliance`
```
Comment (jfagoagas) [inline docs/tutorials/dashboard.md]: Can you elaborate this more?

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: docs/tutorials/dashboard.md (hunk 3 above)
Comment (pedrooot) [reply docs/tutorials/dashboard.md]: Done!

File: docs/tutorials/dashboard.md (hunk 2 above)
Comment (pedrooot) [reply docs/tutorials/dashboard.md]: Done!

**Review by jfagoagas — APPROVED** [review]
Great addition @pedrooot 👏

**Pull request merged**